CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 # Changelog
+### 2.28.0
+This release includes:
+* Fluent Bit [1.9.7](https://fluentbit.io/announcements/v1.9.7/)
```

Comment:
You need to change the Dockerfile and Dockerfile.debug to build the new version, its at the top of the file as a constant